linux_os/guide/system/software/updating/ensure_gpgcheck_globally_activated/oval/shared.xml

```diff
@@ -1,18 +1,42 @@
 <def-group>
+
   <definition class="compliance" id="ensure_gpgcheck_globally_activated" version="1">
     {{{ oval_metadata("The gpgcheck option should be used to ensure that checking
       of an RPM package's signature always occurs prior to its
       installation.") }}}
-    <criteria operator="AND">
-     <criterion comment="check value of gpgcheck in {{{ pkg_manager_config_file }}}" test_ref="test_ensure_gpgcheck_globally_activated" />
-    </criteria>
+    {{% if 'sle' in product %}} 
+      <criteria operator="AND">
+        <criterion comment="check value of gpgcheck in {{{ pkg_manager_config_file }}}" test_ref="test_ensure_gpgcheck_globally_activated" />
+        <criterion comment="no value 0_off_no_false set in {{{ pkg_manager_config_file }}}" test_ref="test_ensure_gpgcheck_globally_no_deactivated" />
```

Comment:
Can we have a test scenario where the first condition is met  but the second isn't met?